src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -378,18 +404,17 @@ class GroupReadsByUmi
   @arg(flag='T', doc="The output tag for UMI grouping.") val assignTag: String = "MI",
   @arg(flag='m', doc="Minimum mapping quality.")         val minMapQ: Int      = 30,
   @arg(flag='n', doc="Include non-PF reads.")            val includeNonPfReads: Boolean = false,
-  @arg(flag='s', doc="The UMI assignment strategy; one of 'identity', 'edit', 'adjacency' or 'paired'.") val strategy: String,
-  @arg(flag='e', doc="The allowable number of edits between UMIs.") val edits: Int = 1
+  @arg(flag='s', doc="The UMI assignment strategy.") val strategy: Strategy,
```

Comment:
This is going to create an annoying compatibility issue, since the tool previously took all-lower-case values for strategy, and now it will only accept title-case.

Reply:
Have you even tried it? It should be case insensitive!

Reply:
Huh, TIL.